src/Kunstmaan/NodeBundle/EventListener/SlugListener.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Kunstmaan\NodeBundle\EventListener;
+
+use Kunstmaan\NodeBundle\Controller\SlugActionInterface;
+use Kunstmaan\NodeBundle\Event\Events;
+use Kunstmaan\NodeBundle\Event\SlugSecurityEvent;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpKernel\Event\FilterControllerEvent;
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\HttpKernel\Controller\ControllerResolver;
+
+
+/**
+ * Class SlugListener
+ * @package Kunstmaan\NodeBundle\EventListener
+ */
+class SlugListener
+{
+
+    /**
+     * @var EntityManager
+     */
+    protected $em;
+
+    /**
+     * @var ControllerResolver
+     */
+    protected $resolver;
+
+
+    /**
+     * @var EventDispatcher
+     */
+    protected $eventDispatcher;
+
+    /**
+     * @param EntityManager $entityManager
+     * @param ControllerResolver $resolver
+     * @param EventDispatcherInterface $eventDispatcher
+     */
+    public function __construct(EntityManager $entityManager, ControllerResolver $resolver, EventDispatcherInterface $eventDispatcher)
+    {
+        $this->em = $entityManager;
+        $this->resolver = $resolver;
+        $this->eventDispatcher = $eventDispatcher;
+    }
+
+    /**
+     * @param FilterControllerEvent $event
+     */
+    public function onKernelController(FilterControllerEvent $event)
+    {
+
+        //check if the event has anything to do with nodeTranslations, if not this method can be skipped
+        if (!$event->getRequest()->attributes->has('_nodeTranslation')) {
+            return;
+        }
+
+        $request = $event->getrequest();
```

Comment:
Change getrequest() to getRequest(), and move this line up, before the if. Use $request in the if (makes the code a tiny bit more readable).